packages/perseus/src/widgets/expression.tsx

```diff
@@ -537,32 +554,44 @@ export class Expression extends React.Component<Props, ExpressionState> {
         );
     };
 
-    render():
-        | React.ReactNode
-        | React.ReactElement<React.ComponentProps<"div">> {
```

Comment:
Ooo, I'm curious why did we need to remove the typing of the render function. If possible I would love to bring it back.

Reply:
<img width="379" alt="Screenshot 2024-07-26 at 8 41 25 AM" src="https://github.com/user-attachments/assets/0ae25db4-8939-4a94-8bfa-f7d2da1db4d5">

`render` gets typed as a result of Expression being typed as `React.Component` so explicit typing is redundant: `class Expression extends React.Component<Props, ExpressionState>`.

But I doubt I would have removed that without having hit an error, I just don't remember what it is.